featuretools/tests/primitive_tests/test_agg_feats.py

```diff
@@ -634,6 +634,35 @@ def test_stacking_multi(es):
         assert fm[cols[i]].tolist() == correct_vals[i] or fm[cols[i]].tolist() == correct_vals1[i]
 
 
+def test_use_previous_pd_dateoffset(es):
+    total_events_last_20_years = ft.Feature(es["log"]["id"],
+                                            parent_entity=es["customers"],
+                                            use_previous="20 years",
+                                            primitive=Count)
+
+    feature_matrix = ft.calculate_feature_matrix([total_events_last_20_years], es)
+
+    total_events_last_20_years_pd = ft.Feature(es["log"]["id"],
+                                               parent_entity=es["customers"],
+                                               use_previous=pd.DateOffset(years=20),
+                                               primitive=Count)
+
+    feature_matrix_2 = ft.calculate_feature_matrix([total_events_last_20_years_pd], es)
+
+    assert (feature_matrix["COUNT(log, Last 20 Years)"] == [10, 5, 2]).all()
+    assert (feature_matrix["COUNT(log, Last 20 Years)"] == feature_matrix_2["COUNT(log, Last 20 Years)"]).all()
+
+    since_str = (pd.to_datetime('today') - pd.tseries.offsets.BDay(100))
+    total_events_last_100_bday = ft.Feature(es["log"]["id"],
+                                            parent_entity=es["customers"],
+                                            use_previous=pd.tseries.offsets.BDay(100),
+                                            primitive=Count)
+    since_str = since_str.strftime('%Y-%m-%d %H:%M:%S')
+    feature_matrix_3 = ft.calculate_feature_matrix([total_events_last_100_bday], es)
+
+    assert (feature_matrix_3["COUNT(log, Last 100 businessdays)".format(since_str)] == [0.0, 0.0, 0.0]).all()
```

Comment:
What exactly is `since_str` replacing in `"COUNT(log, Last 100 businessdays)"`?

Reply:
Would it be possible to choose a unit for business days that don't result in no results to confirm that it works?